python/tvm/relay/op/tensor.py

```diff
@@ -63,6 +63,25 @@ def sqrt(data):
     return _make.sqrt(data)
 
 
+def rsqrt(data):
+    """Compute elementwise rsqrt of data.
+
+    .. math::
+      1/sqrt(x)
```

Comment:
rst requires a new blank line between .. math:: and the formula

Reply:
Thank you. I fixed.